pyDeltaRCM/init_tools.py

```diff
@@ -579,6 +574,19 @@ def _create_meta_variable(varname, varvalue, varunits,
             _create_meta_variable('h0', self.h0, 'meters')
             _create_meta_variable('cell_type', self.cell_type, 'type',
                                   vartype='i8', vardims=('length', 'width'))
+            # subsidence metadata
+            if self._toggle_subsidence:
+                _create_meta_variable('toggle_subsidence',
+                                      self.toggle_subsidence, 'bool')
+                _create_meta_variable('start_subsidence',
+                                      self.start_subsidence, 'seconds',
+                                      vartype='i8')
+                _create_meta_variable('subsidence_rate',
+                                      self.subsidence_rate,
+                                      'meters per second')
+            else:
+                _create_meta_variable('toggle_subsidence',
+                                      self.toggle_subsidence, 'bool')
```

Comment:
why do we need to save the value of `toggle_subsidence`? Since we've removed a number of other variables, can we just record the value of `sigma` as a `LxW` field in the netcdf? 

Definitely can move the `toggle_subsidence` saving outside the if-else though, since it's the same in both logic branches.

Reply:
> why do we need to save the value of toggle_subsidence? Since we've removed a number of other variables, can we just record the value of sigma as a LxW field in the netcdf?

That's a good point, I'll skip the `toggle_subsidence` and just save `start_subsidence` and `sigma` so we know when subsidence begins, and spatially where it happens and at what rate.